catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -129,12 +139,16 @@ public ContentDirectoryMonitor(
       AttributeRegistry attributeRegistry,
       int maxRetries,
       int delayBetweenRetries,
-      Executor configurationExecutor) {
+      Executor configurationExecutor,
+      CdmServiceWaiter cdmServiceWaiter,
+      CdmState initState) {
```

Comment:
✏️ Why is the initState a parameter here? Shouldn't it always be `new CdmState()`?